Source/Applications/openXDA/openXDA/wwwroot/SPCTools/Scripts/TSX/Wizard/WizardTest.tsx

```diff
@@ -234,71 +232,118 @@ const WizardTest = () => {
-                                        { key: 'Threshhold', label: 'Threshhold', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' }, content: (item) => (item.Threshhold == undefined ? "N/A" : item.Threshhold) },
-                                        { key: 'NumberRaised', label: 'Raised', field: 'NumberRaised', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' } },
-                                        { key: 'TimeInAlarm', label: 'Time in Alarm (%)', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' }, content: (item) => item.TimeInAlarm.toFixed(2) + "%" },
-                                    ]}
-                                    tableClass="table thead-dark table-striped"
-                                    data={resultSummary}
-                                    sortKey={'Severity'}
-                                    ascending={false}
-                                    onSort={() => {/* do nothing */}}
-                                    theadStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%' }}
-                                    tbodyStyle={{ display: 'block', overflowY: 'scroll', maxHeight: 240, width: '100%' }}
-                                    rowStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%' }}
-                                    selected={(item) => false}
-                                />
+                                <ReactTable.Table<IResultTable>
+                                    TableClass="table thead-dark table-striped"
+                                    TableStyle={{ maxHeight: '300px' }}
+                                    TheadStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%' }}
+                                    TbodyStyle={{ display: 'block', overflowY: 'scroll', maxHeight: 240, width: '100%' }}
+                                    RowStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%' }}
+                                    Data={resultSummary}
+                                    SortKey={''}
+                                    Ascending={false}
+                                    OnSort={() => {/* do nothing */ }}
+                                    Selected={(item) => false}
+                                    KeySelector={(key) => key.NumberRaised}
+                                >
+                                    <ReactTable.Column
+                                        Key='Severity'
+                                        Field={'Severity'}
+                                        Content={({ item }) => {
+                                            return <p style={{ color: severities.find(s => s.Name == item.Severity).Color }}>{item.Severity}</p>
+                                        }}
+                                        AllowSort={true}
```

Comment:
Would it not be best to set the `AllowSort` flag to `false` if we're preventing sorting?

Reply:
I think that would be wise.

Reply:
Rebased to #360 and retargeted PR